lasy/profiles/from_insight_file.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import h5py
+from scipy.constants import c
+from .from_array_profile import FromArrayProfile
+
+
+class FromInsightFile(FromArrayProfile):
+    r"""
+    Profile defined from insight measurement.
+
+    Parameters
+    ----------
+    file_path: string
+        Path to the file created by insight which contains full field (e.g. Exyt_0.h5)
+
+    pol : list of 2 complex numbers (dimensionless)
+        Polarization vector. It corresponds to :math:`p_u` in the above
+        formula ; :math:`p_x` is the first element of the list and
+        :math:`p_y` is the second element of the list. Using complex
+        numbers enables elliptical polarizations.
+
+    omega0: string or float
+        Set central frequency for the envelope constrauction. Can be a float value
+        in [1/s], or a string defining the method for automatic frequency detection:
+        "barycenter" frequency is averaged over the power profile, "peak" frequency
+        corresponding to the peak field.
```

Comment:
Minor typos fixed, could you check that this is still correct?
```suggestion
    omega0: string or float
        Set the central frequency for the envelope construction. Can be a float value
        in [rad/s], or a string defining the method for automatic frequency detection:
        "barycenter" frequency is averaged over the power profile, "peak" frequency
        corresponding to the location of the maximum of the on-axis spectrum.
```